index.js

```diff
@@ -52,11 +54,28 @@ function neatInput (opts) {
   function handle (ch, key) {
     if (key && key.ctrl) {
       if (key.name === 'c' && !input.emit('ctrl-c')) process.exit()
+      if (key.name === 'b') moveCursorLeft()
+      else if (key.name === 'f') moveCursorRight()
+      else if (key.name === 'a') input.cursor = 0
+      else if (key.name === 'e') input.cursor = rawLine.length
+      else if (key.name === 'u') set('')
+      else if (key.name === 'w') deleteWordBackward()
+      else if (key.name === 'k') rawLine = rawLine.slice(0, input.cursor)
       input.emit('ctrl-' + key.name)
       return true
     }
+
     if (key && key.meta) {
       input.emit('alt-' + key.name)
+      if (key.name === 'b') {
+        input.cursor = util.findWordBeginBackward(rawLine, input.cursor)
+      } else if (key.name === 'f') {
+        input.cursor = util.findWordEndForward(rawLine, input.cursor)
+      } else if (key.name === 'backspace') {
+        deleteWordBackward()
+      } else if (key.name === 'd') {
+        deleteWordForward()
+      }
```

Comment:
I kept the same `else if` style here since the first ones are really long. Let me know if I should change this.